tests/test_onnx.py

```diff
@@ -0,0 +1,7 @@
+import unittest
+
+import onnx
+
+class TestOnnx(unittest.TestCase):
+    def test_load(self):
+        model = onnx.load("data/mnist-8.onnx")
```

Comment:
Can you add an assertion here. Maybe assert on the number of layers or something like that.

Reply:
Asserting some of these data:

![Screen Shot 2022-06-14 at 4 47 37 PM](https://user-images.githubusercontent.com/818310/173685796-1d733f4e-fd9f-4b96-91ed-e4e6f3ecf43d.png)